Bottom: c99c995abee2e9fcd1817d712da3b08d85d66c75
Top:    425a524650f1a11ddeec3c1e033e983fa9617717
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 05:39:56 +0100

Refresh of convert-board-objects-to-glist

---

diff --git a/src/misc.c b/src/misc.c
index bb7b84c..0e25089 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -1696,11 +1696,8 @@ GetGridLockCoordinates (int type, void *ptr1,
       *y = ((ElementTypePtr) ptr2)->MarkY;
       break;
     case POLYGON_TYPE:
-      {
-        PointType *point0 = ((PolygonTypePtr) ptr2)->Points->data;
-        *x = point0->X;
-        *y = point0->Y;
-      }
+      *x = ((PolygonTypePtr) ptr2)->Points[0].X;
+      *y = ((PolygonTypePtr) ptr2)->Points[0].Y;
       break;
 
     case LINEPOINT_TYPE:
